docs/template_plugin/src/template_pattern_transformation.hpp

```diff
@@ -12,20 +12,21 @@
 namespace ngraph {
 namespace pass {
 
-class MyPatternBasedTransformation;
+class DecomposeDivideMatcher;
+class ReluReluFusionMatcher;
 
 }  // namespace pass
 }  // namespace ngraph
 
 // ! [graph_rewrite:template_transformation_hpp]
 // template_pattern_transformation.hpp
-class ngraph::pass::MyPatternBasedTransformation: public ngraph::pass::GraphRewrite {
+class ngraph::pass::DecomposeDivideMatcher: public ngraph::pass::MatcherPass {
```

Comment:
`MatcherPass` -> `PatternMatcherPass`?

Reply:
In nGraph we use Matcher class to store pattern and perform matching on node. So that's why I chose MatcherPass as a name for this class. May be it's better to have smth like PatternPass? @gladilov-gleb what do you think?